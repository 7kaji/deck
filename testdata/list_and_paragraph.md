# List and paragraph

---

# Title

- A
- B

<br><br>

**C**
D
E<br>*F*

- G
- H

