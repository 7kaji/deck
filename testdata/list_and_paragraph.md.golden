[
  {
    "layout": "title-and-body",
    "titles": [
      "List and paragraph"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "A"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "B"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "\n"
              },
              {
                "value": "\n"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "C ",
                "bold": true
              },
              {
                "value": "D "
              },
              {
                "value": "E"
              },
              {
                "value": "\n"
              },
              {
                "value": "F",
                "italic": true
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "G"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "H"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ]
  }
]